content/en/contribute/code/core/deploy-on-eks.md

```diff
@@ -216,7 +216,7 @@ And then follow these steps:
 
    ```shell
    kubectl config use-context arn:aws:eks:eu-west-2:720541322708:cluster/prod-cht-eks
-   ./troubleshooting/get-volume-binding <DEPLOYMENT> <COUCH-DB-NAME> | jq '.subPath'`
+   ./troubleshooting/get-volume-binding <DEPLOYMENT> <COUCH-DB-NAME> | jq '.subPath'
```

Comment:
When I first went through this tutorial, I was confused about where I needed to be at this point. I had only read the cloning section from the document, which didn't mention that I needed to be in a specific folder to run certain commands. We only mention cht-deploy in passing earlier saying that we need it to deploy. 
So I think it would be helpful to mention here that `troubleshooting` are commands from the cht-deploy script location.